bbc/simorgh
# Update Somali and Urdu navigation  to add US election topic

Resolves JIRA n/a

Overall changes
======
Adds the US election topic link to the navigation bar of Somali and Urdu



Code changes
======
- Edited Navigation section of src/app/lib/config/services/somali.ts to add election topic link in second position
- Edited Navigation section of src/app/lib/config/services/urdu.ts to add election topic link in second positiontion
- Updated snapshots



Testing
======
1. Open Somali's front page, second link in the nav should be Doorashada Mareykanka 2024 and open the US election topic
2. Open Urdu's front page, second link in the nav should be امریکی صدارتی انتخاب 2024 and open the US election topic




<img width="574" alt="somali_nav" src="https://github.com/user-attachments/assets/a42ccf0f-e15c-4625-9d31-6247d45a9cd7">
<img width="822" alt="urdu_nav" src="https://github.com/user-attachments/assets/ec65adf5-9b86-401f-9859-0587207bff5d">


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
